.github/workflows/fastembed.yml

```diff
@@ -43,7 +43,9 @@ jobs:
 
       - name: Run tests
         id: tests
-        run: hatch run cov
+        run: |
+          hatch run pip install git+https://github.com/deepset-ai/haystack.git #TODO: rm before merging
+          hatch run cov        
```

Comment:
@lambda-science I temporarily modified the workflow to install Haystack from main.
This way we can experiment...

Reply:
PR is ready I guess, it's the last thing to do :)